libraries/chain/include/eosio/chain/snapshot_detail.hpp

```diff
@@ -135,6 +136,7 @@ namespace eosio::chain::snapshot_detail {
          , proposer_policies(bs.proposer_policies)
          , finalizer_policies(bs.finalizer_policies)
          , finalizer_policy_generation(bs.finalizer_policy_generation)
+         , last_pending_finalizer_policy_digest(bs.last_pending_finalizer_policy_digest)
```

Comment:
Should we add a comment on top of this function:

When adding a member initialization here, also update `block_state::block_state(snapshot_detail::snapshot_block_state_v7&& sbs)`.